src/main/webapp/app/exercises/programming/shared/lifecycle/programming-exercise-lifecycle.component.html

```diff
@@ -138,6 +138,24 @@ <h6 jhiTranslate="artemisApp.assessment.assessment"></h6>
                 <jhi-help-icon placement="right auto" [text]="'artemisApp.programmingExercise.timeline.complaintOnAutomaticAssessmentTooltip'" />
             </div>
         }
+        @if (isExamMode && isImport && importOptions) {
+            <div class="form-check">
+                <input
+                    type="checkbox"
+                    class="form-check-input"
+                    name="setTestCaseVisibilityToAfterDueDate"
+                    [checked]="importOptions.setTestCaseVisibilityToAfterDueDate"
+                    id="setTestCaseVisibilityToAfterDueDate-checkbox"
+                    (change)="importOptions.setTestCaseVisibilityToAfterDueDate = !importOptions.setTestCaseVisibilityToAfterDueDate"
```

Comment:
personal preference: I would create a toggle-function. Makes the html file bit cleaner. :)